merlin-server/constraints-dsl-compiler/package.json

```diff
@@ -8,7 +8,7 @@
     "test": "npm run build && node build/main.js",
     "generate-ast-schema": "npx typescript-json-schema src/libs/constraints-ast.ts \"*\" --aliasRefs --out ./build/constraints.schema.json",
     "reformat": "npx prettier --config ./.prettierrc -w ./src",
-    "generate-doc": "npx typedoc src/libs/constraints-edsl-fluent-api.ts --excludePrivate --excludeInternal --out ./build/docs --name \"Constraints eDSL documentation\""
+    "generate-doc": "npx typedoc src/libs/constraints-edsl-fluent-api.ts --excludePrivate --excludeInternal --out ./build/docs/constraints-dsl-compiler --name \"Constraints eDSL documentation\""
```

Comment:
For both here and in `scheduler-server/scheduling-dsl-compiler/package.json`, why was this change made?

Reply:
This puts it in a subdirectory - which we need so we don't have various files stomping on each other when we add them into the docsite with `html_extra_files`.